src/_buttons.scss

```diff
@@ -225,3 +225,78 @@
     }
   }
 }
+
+
+.button.is-icon-only-button {
```

Comment:
This class should be a modifier on `.button`, i.e. `class="button button__icon-only"`. `is-*` should be reserved for atomic modifications to default styles, such as size or color.